### ISSUE TITLE
Not generating pseudo replicates for tag replicates

### ISSUE THREAD

**Issue by praneet007** [karmel/homer-idr#4]
Hello,
I have chip-seq data from xenopus which has basically scaffolds instead of chromosomes and thus while making tag directories using homer i gave the option -single as there are more than thousands of scaffolds.
Now when I am running IDR analysis its like I am not able to generate pseudoreps using run_idr.py.

Here is the error::
Creating directory: Pseudoreplicates_individual/St20_Bcat_1-Pseudorep1 and removing existing *.tags.tsv
	Adding tag directory Pseudoreplicates_individual/St20_Bcat_1-Pseudorep1-tmp
	!!! Something is wrong - no reads were added to tag directory !!!
	!!! Check your input files or the makeTagDirectory command options... !!!
	Will add tag directory: Pseudoreplicates_individual/St20_Bcat_1-Pseudorep2-tmp

**Comment by karmel** [comment]
Are you sure there are tags output in the original Homer directory? If you list that tags directory, how big are the files? Can you include the commands and output used up to this point? Thanks!

**Comment by praneet007** [comment]
Hello,
Hope you are doing fine. Here I am attaching the snap of how directory
looks like in my case. Also I managed pseudo rep step using the original
mapped bed files. But when I running last step of run_idr.py idr , with 4
replicates it ran successfully but in case of three it gave the error
attached.

Thanks,

Regards,
Praneet Chaturvedi

**Comment by onkarnath89** [comment]
I am also facing the same issue with my chip-seq data. I have tag files of ~100mbs. but when I run run_idr.py it produces this error saying "Something is wrong - no reads were added to tag directory".
I tried creating tags with different options but none succeeded in idr run or in pseudoreplicate generation using run_idr.py.
Do anyone have quick suggestions for this issue.

@Homer @cbenner

**Comment by karmel** [comment]
The most likely cause is a file pathing issue-- can you post the commands you used up until this point, and the locations of the files being used as input?

**Comment by onkarnath89** [comment]
mkdir ~/MJ_work-HIDR
cd ~/MJ_work-HIDR
mkdir read_file
cd read_file/
cp ~/MJ_work-1-2/MJChip00?_filtered.fastq .
cd ..
cp ~/MJ_work-1-2/MJChip00?_filtered.bam read_file/
makeTagDirectory MJ_work-Input-Combined read_file/MJChip003_filtered.bam read_file/MJChip004_filtered.bam 
makeTagDirectory MJ_work-chip-Combined read_file/MJChip001_filtered.bam read_file/MJChip002_filtered.bam 
mkdir -p peaks/replicates
cd peaks/replicates
findPeaks ../../MJ_work-chip-Combined -P .1 -LP .1 -poisson .1 -style factor -nfr -i ~/MJ_work-HIDR/MJ_work-Input-Combined -o MJChip001_filtered_peaks.txt
for i in 1 2 3 4; do makeTagDirectory MJ_chip00${i} read_file/MJChip00${i}_filtered.bam; done
cd peaks/replicates/
findPeaks ../../MJ_chip001 -P .1 -LP .1 -poisson .1 -style factor -nfr -i ~/MJ_work-HIDR/MJ_work-Input-Combined -o MJChip001_filtered_peaks.txt
findPeaks ../../MJ_chip002 -P .1 -LP .1 -poisson .1 -style factor -nfr -i ~/MJ_work-HIDR/MJ_work-Input-Combined -o MJChip002_filtered_peaks.txt
mkdir -p pseudoreps/individual
python ~/Softwares/homer-idr/homer-idr/idr/run_idr.py pseudoreplicate -d MJ_chip001_2/ -o pseudoreps/individual/

At This step I got the error message:
Generating 2 pseudoreplicate tag directories for MJ_chip001_2
    Will add tag directory: pseudoreps/individual/MJ_chip001_2-Pseudorep1-tmp
    Creating directory: pseudoreps/individual/MJ_chip001_2-Pseudorep1 and removing existing *.tags.tsv
    Adding tag directory pseudoreps/individual/MJ_chip001_2-Pseudorep1-tmp
    !!! Something is wrong - no reads were added to tag directory !!!
    !!! Check your input files or the makeTagDirectory command options... !!!
    Will add tag directory: pseudoreps/individual/MJ_chip001_2-Pseudorep2-tmp
    Creating directory: pseudoreps/individual/MJ_chip001_2-Pseudorep2 and removing existing *.tags.tsv
    Adding tag directory pseudoreps/individual/MJ_chip001_2-Pseudorep2-tmp
    !!! Something is wrong - no reads were added to tag directory !!!
    !!! Check your input files or the makeTagDirectory command options... !!!

I tried building tagdirectories some other parameters even but the eroor remained the same.

**Comment by karmel** [comment]
It's hard to follow in this format, but you don't seem to be returning to the top directory as often as the instructions would require-- can you also post the directory structure that you ended up with there? (Not the individual files, but at least the tree structure of all the directories.)

**Comment by onkarnath89** [comment]
**Working Directory:**
/storage/onkar/MJ_work-HIDR

**Tree structure:**
|-- idr-output
|   |-- narrowpeaks
|   |   |-- MJChip001_filtered_peaks_273.narrowPeak
|   |   |-- MJChip001_filtered_peaks_273-truncated.narrowPeak
|   |   |-- MJChip002_filtered_peaks_25.narrowPeak
|   |   `-- MJChip002_filtered_peaks_25-truncated.narrowPeak
|   |-- plots
|   |`-- Replicate_comparison-plot.ps
|   |-- pooled_comparisons
|   |   |-- MJChip001_filtered_peaks_273-truncated-pair-em.sav
|   |   |-- MJChip001_filtered_peaks_273-truncated-pair-npeaks-aboveIDR.txt
|   |   |-- MJChip001_filtered_peaks_273-truncated-pair-overlapped-peaks.txt
|   |   |-- MJChip001_filtered_peaks_273-truncated-pair-Rout.txt
|   |   `-- MJChip001_filtered_peaks_273-truncated-pair-uri.sav
|   |-- pseudorep_comparisons
|`-- replicate_comparisons
|       |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-em.sav
|       |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-npeaks-aboveIDR.txt
|       |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-overlapped-peaks.txt
|       |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-Rout.txt
|       `-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-uri.sav
|-- MJ_chip001
|   |-- Chr10.tags.tsv
|   |-- Chr11.tags.tsv
|   |-- Chr12.tags.tsv
|   |-- Chr1.tags.tsv
|   |-- Chr2.tags.tsv
|   |-- Chr3.tags.tsv
|   |-- Chr4.tags.tsv
|   |-- Chr5.tags.tsv
|   |-- Chr6.tags.tsv
|   |-- Chr7.tags.tsv
|   |-- Chr8.tags.tsv
|   |-- Chr9.tags.tsv
|   |-- ChrSy.tags.tsv
|   |-- ChrUn.tags.tsv
|   |-- MJChip001_filtered.bam
|   |-- MJChip001_filtered.bed
|   |-- tagAutocorrelation.txt
|   |-- tagCountDistribution.txt
|   |-- tagInfo.txt
|`-- tagLengthDistribution.txt
|-- MJ_chip002
|   |-- Chr10.tags.tsv
|   |-- Chr11.tags.tsv
|   |-- Chr12.tags.tsv
|   |-- Chr1.tags.tsv
|   |-- Chr2.tags.tsv
|   |-- Chr3.tags.tsv
|   |-- Chr4.tags.tsv
|   |-- Chr5.tags.tsv
|   |-- Chr6.tags.tsv
|   |-- Chr7.tags.tsv
|   |-- Chr8.tags.tsv
|   |-- Chr9.tags.tsv
|   |-- ChrSy.tags.tsv
|   |-- ChrUn.tags.tsv
|   |-- MJChip002_filtered.bam
|   |-- tagAutocorrelation.txt
|   |-- tagCountDistribution.txt
|   |-- tagInfo.txt
|   `-- tagLengthDistribution.txt
|-- MJ_chip003
|   |-- Chr10.tags.tsv
|   |-- Chr11.tags.tsv
|   |-- Chr12.tags.tsv
|   |-- Chr1.tags.tsv
|   |-- Chr2.tags.tsv
|   |-- Chr3.tags.tsv
|   |-- Chr4.tags.tsv
|   |-- Chr5.tags.tsv
|   |-- Chr6.tags.tsv
|   |-- Chr7.tags.tsv
|   |-- Chr8.tags.tsv
|   |-- Chr9.tags.tsv
|   |-- ChrSy.tags.tsv
|   |-- ChrUn.tags.tsv
|   |-- tagAutocorrelation.txt
|   |-- tagCountDistribution.txt
|   |-- tagInfo.txt
|`-- tagLengthDistribution.txt
|-- MJ_chip004
|   |-- Chr10.tags.tsv
|   |-- Chr11.tags.tsv
|   |-- Chr12.tags.tsv
|   |-- Chr1.tags.tsv
|   |-- Chr2.tags.tsv
|   |-- Chr3.tags.tsv
|   |-- Chr4.tags.tsv
|   |-- Chr5.tags.tsv
|   |-- Chr6.tags.tsv
|   |-- Chr7.tags.tsv
|   |-- Chr8.tags.tsv
|   |-- Chr9.tags.tsv
|   |-- ChrSy.tags.tsv
|   |-- ChrUn.tags.tsv
|   |-- tagAutocorrelation.txt
|   |-- tagCountDistribution.txt
|   |-- tagInfo.txt
|   `-- tagLengthDistribution.txt
|-- MJ_RICE_work-chip-Combined
|   |-- Chr10.tags.tsv
|   |-- Chr11.tags.tsv
|   |-- Chr12.tags.tsv
|   |-- Chr1.tags.tsv
|   |-- Chr2.tags.tsv
|   |-- Chr3.tags.tsv
|   |-- Chr4.tags.tsv
|   |-- Chr5.tags.tsv
|   |-- Chr6.tags.tsv
|   |-- Chr7.tags.tsv
|   |-- Chr8.tags.tsv
|   |-- Chr9.tags.tsv
|   |-- ChrSy.tags.tsv
|   |-- ChrUn.tags.tsv
|   |-- tagAutocorrelation.txt
|   |-- tagCountDistribution.txt
|   |-- tagInfo.txt
|`-- tagLengthDistribution.txt
|-- MJ_RICE_work-Input-Combined
|   |-- Chr10.tags.tsv
|   |-- Chr11.tags.tsv
|   |-- Chr12.tags.tsv
|   |-- Chr1.tags.tsv
|   |-- Chr2.tags.tsv
|   |-- Chr3.tags.tsv
|   |-- Chr4.tags.tsv
|   |-- Chr5.tags.tsv
|   |-- Chr6.tags.tsv
|   |-- Chr7.tags.tsv
|   |-- Chr8.tags.tsv
|   |-- Chr9.tags.tsv
|   |-- ChrSy.tags.tsv
|   |-- ChrUn.tags.tsv
|   |-- tagAutocorrelation.txt
|   |-- tagCountDistribution.txt
|   |-- tagInfo.txt
|   `-- tagLengthDistribution.txt
|-- peaks
|   |-- pooled
|   |`-- MJ_RICE_work-chip-Combined_peaks.txt
|   `-- replicates
|       |-- MJChip001_filtered_peaks.txt
|`-- MJChip002_filtered_peaks.txt
|-- pseudoreps
|   `-- individual
|       |-- MJ_chip001
|       |   |-- MJChip001_filtered.bed-Pseudorep1-tmp
|       |   |-- MJChip001_filtered.bed-Pseudorep2-tmp
|       |   |-- MJ_chip001-Pseudorep1
|       |`-- MJ_chip001-Pseudorep2
|       |-- MJ_chip001_1-Pseudorep1
|       |-- MJ_chip001_1-Pseudorep2
|       |-- MJ_chip001_2-Pseudorep1
|       |-- MJ_chip001_2-Pseudorep2
|       |-- MJ_chip001_3-Pseudorep1
|       |-- MJ_chip001_3-Pseudorep2
|       |-- MJ_chip001_4-Pseudorep1
|       |-- MJ_chip001_4-Pseudorep2
|       |-- MJ_chip001_5-Pseudorep1
|       |-- MJ_chip001_5-Pseudorep2
|       |-- MJ_chip001-Pseudorep1
|       |-- MJ_chip001-Pseudorep2
|       |-- MJ_chip002-Pseudorep1
|       `-- MJ_chip002-Pseudorep2
`-- read_file
    |-- MJChip001_filtered.bam
    |-- MJChip001_filtered.fastq
    |-- MJChip002_filtered.bam
    |-- MJChip002_filtered.fastq
    |-- MJChip003_filtered.bam
    |-- MJChip003_filtered.fastq
    |-- MJChip004_filtered.bam
    `-- MJChip004_filtered.fastq

**Comment by karmel** [comment]
It took some parsing-- but I think this is what the above represents:

```
|-- idr-output
| |-- narrowpeaks
| | |-- MJChip001_filtered_peaks_273.narrowPeak
| | |-- MJChip001_filtered_peaks_273-truncated.narrowPeak
| | |-- MJChip002_filtered_peaks_25.narrowPeak
| | -- MJChip002_filtered_peaks_25-truncated.narrowPeak 
| |-- plots 
| |-- Replicate_comparison-plot.ps
| |-- pooled_comparisons
| | |-- MJChip001_filtered_peaks_273-truncated-pair-em.sav
| | |-- MJChip001_filtered_peaks_273-truncated-pair-npeaks-aboveIDR.txt
| | |-- MJChip001_filtered_peaks_273-truncated-pair-overlapped-peaks.txt
| | |-- MJChip001_filtered_peaks_273-truncated-pair-Rout.txt
| | -- MJChip001_filtered_peaks_273-truncated-pair-uri.sav 
| |-- pseudorep_comparisons 
|-- replicate_comparisons
| |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-em.sav
| |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-npeaks-aboveIDR.txt
| |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-overlapped-peaks.txt
| |-- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-Rout.txt
| -- MJChip001_filtered_peaks_273-truncated-MJChip002_filtered_peaks_25-truncated-uri.sav 
|-- MJ_chip001 
| |-- Chr10.tags.tsv
| |-- Chr11.tags.tsv
| |-- Chr12.tags.tsv
| |-- Chr1.tags.tsv
| |-- Chr2.tags.tsv
| |-- Chr3.tags.tsv
| |-- Chr4.tags.tsv
| |-- Chr5.tags.tsv
| |-- Chr6.tags.tsv
| |-- Chr7.tags.tsv
| |-- Chr8.tags.tsv
| |-- Chr9.tags.tsv
| |-- ChrSy.tags.tsv
| |-- ChrUn.tags.tsv
| |-- MJChip001_filtered.bam
| |-- MJChip001_filtered.bed
| |-- tagAutocorrelation.txt
| |-- tagCountDistribution.txt
| |-- tagInfo.txt 
|-- tagLengthDistribution.txt
|-- MJ_chip002
| |-- Chr10.tags.tsv
| |-- Chr11.tags.tsv
| |-- Chr12.tags.tsv
| |-- Chr1.tags.tsv
| |-- Chr2.tags.tsv
| |-- Chr3.tags.tsv
| |-- Chr4.tags.tsv
| |-- Chr5.tags.tsv
| |-- Chr6.tags.tsv
| |-- Chr7.tags.tsv
| |-- Chr8.tags.tsv
| |-- Chr9.tags.tsv
| |-- ChrSy.tags.tsv
| |-- ChrUn.tags.tsv
| |-- MJChip002_filtered.bam
| |-- tagAutocorrelation.txt
| |-- tagCountDistribution.txt
| |-- tagInfo.txt
| -- tagLengthDistribution.txt 
|-- MJ_chip003
| |-- Chr10.tags.tsv
| |-- Chr11.tags.tsv
| |-- Chr12.tags.tsv
| |-- Chr1.tags.tsv
| |-- Chr2.tags.tsv
| |-- Chr3.tags.tsv
| |-- Chr4.tags.tsv
| |-- Chr5.tags.tsv
| |-- Chr6.tags.tsv
| |-- Chr7.tags.tsv
| |-- Chr8.tags.tsv
| |-- Chr9.tags.tsv
| |-- ChrSy.tags.tsv
| |-- ChrUn.tags.tsv
| |-- tagAutocorrelation.txt
| |-- tagCountDistribution.txt
| |-- tagInfo.txt 
|-- tagLengthDistribution.txt
|-- MJ_chip004
| |-- Chr10.tags.tsv
| |-- Chr11.tags.tsv
| |-- Chr12.tags.tsv
| |-- Chr1.tags.tsv
| |-- Chr2.tags.tsv
| |-- Chr3.tags.tsv
| |-- Chr4.tags.tsv
| |-- Chr5.tags.tsv
| |-- Chr6.tags.tsv
| |-- Chr7.tags.tsv
| |-- Chr8.tags.tsv
| |-- Chr9.tags.tsv
| |-- ChrSy.tags.tsv
| |-- ChrUn.tags.tsv
| |-- tagAutocorrelation.txt
| |-- tagCountDistribution.txt
| |-- tagInfo.txt
| -- tagLengthDistribution.txt 
|-- MJ_RICE_work-chip-Combined
| |-- Chr10.tags.tsv
| |-- Chr11.tags.tsv
| |-- Chr12.tags.tsv
| |-- Chr1.tags.tsv
| |-- Chr2.tags.tsv
| |-- Chr3.tags.tsv
| |-- Chr4.tags.tsv
| |-- Chr5.tags.tsv
| |-- Chr6.tags.tsv
| |-- Chr7.tags.tsv
| |-- Chr8.tags.tsv
| |-- Chr9.tags.tsv
| |-- ChrSy.tags.tsv
| |-- ChrUn.tags.tsv
| |-- tagAutocorrelation.txt
| |-- tagCountDistribution.txt
| |-- tagInfo.txt 
|-- tagLengthDistribution.txt
|-- MJ_RICE_work-Input-Combined
| |-- Chr10.tags.tsv
| |-- Chr11.tags.tsv
| |-- Chr12.tags.tsv
| |-- Chr1.tags.tsv
| |-- Chr2.tags.tsv
| |-- Chr3.tags.tsv
| |-- Chr4.tags.tsv
| |-- Chr5.tags.tsv
| |-- Chr6.tags.tsv
| |-- Chr7.tags.tsv
| |-- Chr8.tags.tsv
| |-- Chr9.tags.tsv
| |-- ChrSy.tags.tsv
| |-- ChrUn.tags.tsv
| |-- tagAutocorrelation.txt
| |-- tagCountDistribution.txt
| |-- tagInfo.txt
| -- tagLengthDistribution.txt
|-- peaks
| |-- pooled
| |-- MJ_RICE_work-chip-Combined_peaks.txt
|-- replicates
| |-- MJChip001_filtered_peaks.txt
|-- MJChip002_filtered_peaks.txt
|-- pseudoreps
|-- individual
| |-- MJ_chip001
| | |-- MJChip001_filtered.bed-Pseudorep1-tmp
| | |-- MJChip001_filtered.bed-Pseudorep2-tmp
| | |-- MJ_chip001-Pseudorep1
| |-- MJ_chip001-Pseudorep2
| |-- MJ_chip001_1-Pseudorep1
| |-- MJ_chip001_1-Pseudorep2
| |-- MJ_chip001_2-Pseudorep1
| |-- MJ_chip001_2-Pseudorep2
| |-- MJ_chip001_3-Pseudorep1
| |-- MJ_chip001_3-Pseudorep2
| |-- MJ_chip001_4-Pseudorep1
| |-- MJ_chip001_4-Pseudorep2
| |-- MJ_chip001_5-Pseudorep1
| |-- MJ_chip001_5-Pseudorep2
| |-- MJ_chip001-Pseudorep1
| |-- MJ_chip001-Pseudorep2
| |-- MJ_chip002-Pseudorep1
| -- MJ_chip002-Pseudorep2 -- read_file
|-- MJChip001_filtered.bam
|-- MJChip001_filtered.fastq
|-- MJChip002_filtered.bam
|-- MJChip002_filtered.fastq
|-- MJChip003_filtered.bam
|-- MJChip003_filtered.fastq
|-- MJChip004_filtered.bam
|-- MJChip004_filtered.fastq
```

In which case it does indeed look like somehow the file structure got very confused. I would suggest restarting according to the instructions, and making sure the final directory structure replicates what is detailed there, with 

```
peaks
- replicates
- pooled
- pseudoreps
- pooled-pseudoreps
pseudoreps
- individual
- pooled
```

**Comment by onkarnath89** [comment]
I repeated this from scratch as per your instructions but still the same problem persists.
when I ran idr it it didn't considered the tag directories.

**python ~/Softwares/homer-idr/homer-idr/idr/run_idr.py pseudoreplicate -d MJ-Chip-1 MJ-Chip-2 -o pseudoreps/individual**
Generating 2 pseudoreplicate tag directories for MJ-Chip-1
    Will add tag directory: pseudoreps/individual/MJ-Chip-1-Pseudorep1-tmp

```
Creating directory: pseudoreps/individual/MJ-Chip-1-Pseudorep1 and removing existing *.tags.tsv
Adding tag directory pseudoreps/individual/MJ-Chip-1-Pseudorep1-tmp
!!! Something is wrong - no reads were added to tag directory !!!
!!! Check your input files or the makeTagDirectory command options... !!!
Will add tag directory: pseudoreps/individual/MJ-Chip-1-Pseudorep2-tmp

Creating directory: pseudoreps/individual/MJ-Chip-1-Pseudorep2 and removing existing *.tags.tsv
Adding tag directory pseudoreps/individual/MJ-Chip-1-Pseudorep2-tmp
!!! Something is wrong - no reads were added to tag directory !!!
!!! Check your input files or the makeTagDirectory command options... !!!
```

Generating 2 pseudoreplicate tag directories for MJ-Chip-2
    Will add tag directory: pseudoreps/individual/MJ-Chip-2-Pseudorep1-tmp

```
Creating directory: pseudoreps/individual/MJ-Chip-2-Pseudorep1 and removing existing *.tags.tsv
Adding tag directory pseudoreps/individual/MJ-Chip-2-Pseudorep1-tmp
!!! Something is wrong - no reads were added to tag directory !!!
!!! Check your input files or the makeTagDirectory command options... !!!
Will add tag directory: pseudoreps/individual/MJ-Chip-2-Pseudorep2-tmp

Creating directory: pseudoreps/individual/MJ-Chip-2-Pseudorep2 and removing existing *.tags.tsv
Adding tag directory pseudoreps/individual/MJ-Chip-2-Pseudorep2-tmp
!!! Something is wrong - no reads were added to tag directory !!!
!!! Check your input files or the makeTagDirectory command options... !!!
```

**Comment by karmel** [comment]
It looks like homer is not finding the tag directories. I would suggest using the full path to them if you are having trouble with pathing.

**Comment by onkarnath89** [comment]
I tried doing that, But that too didn't succeed.

**Comment by karmel** [comment]
Sorry @onkarnath89 , but this is hard to debug without having access to what you are doing. I would suggest going through the instructions exactly as shown, and making sure at each step that your file structure still looks like it should. Make sure your initial tag directories actually have tags, too.

**Comment by Xiaofei-git** [comment]
@praneet007 @onkarnath89 Have you got a solution to the error? I got the same issue on my data, I tired to give the full path but still did not fix it. Any comments are appreciate!

**Comment by onkarnath89** [comment]
@Xiaofei-git No I didn't find any solution for this. So what I did was I left the idea of generating tag directories. 
What I remember I did is I modified the file manually according to the requirement of idr input and then It ran successfully.
But now I am not using idr for my other studies I have moved to other strategy.

**Comment by onkarnath89** [comment]
Most of the softwares that are dependent on tag directory generation, creates problem (due to its dependencies etc). And its never a good option to invest your brain and time solving these issues instead of your real work. So I am not preferring any such tool. Yes if one is very good then I use their strategy manually on my results..

**Comment by Xiaofei-git** [comment]
@onkarnath89 could you share the "other strategy" that you mentioned about biological rep. for chip-seq with me? Thanks!

**Comment by Xiaofei-git** [comment]
@onkarnath89 is it the overlapped peaks between different replicates?

**Comment by onkarnath89** [comment]
@Xiaofei-git yes, these are overlapped peaks. I even tried with non overlapped peaks but still it didn't worked.
And Sorry at present I cannot share our strategy, as its not yet published and validated well. It will be included in our publication. we are waiting for our wet lab validation to complete then it will be published.
You can be updated of our publications at "https://scholar.google.co.in/citations?hl=en&user=O4gzGPUAAAAJ&view_op=list_works&sortby=pubdate"

**Comment by floem7** [comment]
Hi @praneet007 @karmel @onkarnath89 @Xiaofei-git 

I've also encountered the discussed issue.
I think it is file-naming issue, I've tag files in following form, eg. "1.tags.tsv" I see that @onkarnath89 have
"Chr1.tags.tsv", both don't conform naming in `utils.py` script in "bin/homer-idr/idr/" directory.

Take a look

`for f in os.listdir(tag_dir):`
`            # If this is a chr[X].txt file`
`            if re.match('chr[A-Za-z0-9]+\.tags\.tsv$',f):`

In my case the problem is rooted in that HOMER allows some variation in chromosome/scaffolds naming in genome fasta file used to create tag directory (I've used custom maize genome with chromosomes named simply with numbers, plus "MT" and "Pt" for mitochondrial and plastidial, respectively).

In my case renaming files helps, so that pseudoreplicates are generated, I don't know how it will be going further though ... ;)
Alternatively one could delete `chr` string from the code.
I hope it helps someone, so project homer-idr will be revived.

**Comment by Xiaofei-git** [comment]
@floem7 
So, you mean we have to change the chromosome, prefixing with "chr", like chr1, even for "MT" and "Pt" (in my previous case, it is same as, eg. "1.tags.tsv")? I will try it because I will need do analysis on another dataset. 

BTW, did you make it run successfully going further?

Thanks a lot!

Xiaofei

**Comment by floem7** [comment]
@Xiaofei-git

As of renaming files, I though about it but I have decided rather to edit the 47th line of the 
> utils.py

> if re.match('[A-Za-z0-9]+\.tags\.tsv$',f):

Further it ran without problems :-)

FYI I've contacted Anshul Kundaje and he suggested me to use following package:
https://github.com/nboley/idr

Nevertheless, eventually I've decided not to use IDR

**Comment by Xiaofei-git** [comment]
> Nevertheless, eventually I've decided not to use IDR

Is there any inspiration make you decide not t ouse IDR?

Thanks!

**Comment by floem7** [comment]
In fact I'm still not sure if IDR is appropriate in case of FAIRE-seq.
I've searched internet and found (mostly on various forums) posts both for and against.

**Comment by Xiaofei-git** [comment]
Thanks a lot!

**Comment by Xiaofei-git** [comment]
Hey,

I just signed the petition "Stop racial stereotyping show in school!" and
wanted to see if you could help by adding your name.

Our goal is to reach 1,000 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/GgPr6WMbHM

Thanks!
Xiaofei
